support-frontend/assets/components/tooltip/Tooltip.tsx

```diff
@@ -59,7 +59,8 @@ const tooltipContainer = css`
 	}
 `;
 
-const tooltipCss = css`
+const tooltipCss = (hide: boolean) => css`
```

Comment:
I believe screen readers generally ignore anything with `display: none`, so this could affect the accessibility. An alternative is to apply `overflow: hidden` on the `<p>` paragraph's parent `<div>` element, as the issues here is the the paragraph overflows beyond the dimensions of it's parent element. By applying `overflow: hidden` this overflow is clipped.

Reply:
You shouldn't need to apply this rule conditionally either.